graphit_eval/eval/table7/Makefile

```diff
@@ -22,13 +23,13 @@ OPENMP_FLAGS += -DOPENMP -qopenmp -O3
 else ifdef ICPC_TBB
 # compiling with Intel icpc compiler
 PCC = icpc
-CILK_FLAGS += -qopenmp -DTBB -O3 -ltbb
+TBB_FLAGS += -qopenmp -DTBB -O3 -ltbb
 OPENMP_FLAGS += -DOPENMP -qopenmp -O3
 
 else ifdef TBB 
 PCC = g++
-CILK_FLAGS += -I /data/scratch/yunming/intel_parallel_studio_cluster/parallel_studio_xe_2017/install/compilers_and_libraries_2017.2.174/linux/tbb/include/ -L /data/scratch/yunming/intel_parallel_studio_cluster/parallel_studio_xe_2017/install/compilers_and_libraries_2017.2.174/linux/tbb/lib/intel64/gcc4.7/ -fopenmp -DTBB -O3 -ltbb 
-OPENMP_FLAGS += -DOPENMP -fopenmp -O3
+TBB_FLAGS += -I /data/scratch/yunming/intel_parallel_studio_cluster/parallel_studio_xe_2017/install/compilers_and_libraries_2017.2.174/linux/tbb/include/ -L /data/scratch/yunming/intel_parallel_studio_cluster/parallel_studio_xe_2017/install/compilers_and_libraries_2017.2.174/linux/tbb/lib/intel64/gcc4.7/ -fopenmp -DTBB -O3 -ltbb 
+OPENMP_FLAGS += -DOPENMP -qopenmp -O3
```

Comment:
This is probably unnecessary but it was defined like this in ICPC_TBB so didn't want change the styling
@yunmingzhang17

Reply:
I am talking about the OPENMP_FLAGS